src/redux/configureStore.js

```diff
@@ -1,11 +1,10 @@
 import { combineReducers, configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] For this activity, we want you to use `createAsyncThunk` for defining _thunk action creators_, which should be similar to this example:

```javascript
import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
import axios from 'axios'; 


export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 
```

Kindly ensure to improve upon the same!

Reply:
@rloterh I cannot understand your point, can you please elaborate?
I have already used createAsyncThunk in bookstoreapi.js

Reply:
@Wahaj-Ali 
Your project is complete. Kindly resubmit for approval!

Cheers!